GSoC/2016/README.md

```diff
@@ -0,0 +1,29 @@
+# Note for 2016 GSoC application
+
+It's my first time in the GSoC application. Due to my lack of the process, I missed the deadline of updating my enrollment letter, which finally caused the eligibility.
+
+This proposal was not selected as one of the GSoC project as the enrollment letter was not in the correct form. I didn't know this massage until I checked the GSoC website manually before the last day which was too late according to the rules. So unfortunately, this project will not be eligible.
```

Comment:
~~This proposal was not selected as one of the GSoC project as the enrollment letter was not in the correct form.~~ I didn't know the requirement of enrollment letter until I checked the GSoC website one day before the deadline. Unfortunately, this proposal was not approved only because of the miss of deadline in submitting a supporting letter. But the proposal itself has passed organizational peer reviews and could be useful for future reference.